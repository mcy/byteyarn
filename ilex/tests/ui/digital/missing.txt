0xdeadbeef
0x 0xf
