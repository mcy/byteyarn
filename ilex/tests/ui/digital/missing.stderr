error: expected digits after `0x`, but found ` `
 --> digital/missing.txt:2:3
  |
2 | 0x 0xf
  |   ^ expected digits after `0x`
  | ^^ because of this prefix
  |
  = note: reported at: ilex/src/rt/emit2.rs:571:18

error: aborting due to previous error
